### PR TITLE
Fixed: Assume category path from qBittorent starting with '//' is a Windows path

### PR DESCRIPTION
**Pull request by markus101**
#### Description

For category paths qbit returns UNC paths with forward slashes instead of back slashes, this will convert those to backslashes so they can be handled properly by Sonarr. Anyone that incorrectly puts `//` at the start of their *nix path will need to fix them and should feel shame.

#### Issues Fixed or Closed by this PR
* Closes Radarr/Radarr#10162

### CODE REVIEW

**Comment by markus101** [comment]
It's called in 5 different checks and not cached, so could be a little annoying, but added.

File: src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs [hunk 1]
```diff
@@ -381,6 +381,7 @@ public override DownloadClientInfo GetStatus()
 
                     if (savePath.StartsWith("//"))
                     {
+                        _logger.Trace("Replacing slashes in probable Windows UNC path '{0}'. If this is not meant to be a Windows UNC path fix your qBittorrent configuration", savePath);
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs:384]: User hat: fix how?

File: src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs [hunk 2]
```diff
@@ -377,7 +377,15 @@ public override DownloadClientInfo GetStatus()
             {
                 if (Proxy.GetLabels(Settings).TryGetValue(Settings.TvCategory, out var label) && label.SavePath.IsNotNullOrWhiteSpace())
                 {
-                    var labelDir = new OsPath(label.SavePath);
+                    var savePath = label.SavePath;
+
+                    if (savePath.StartsWith("//"))
+                    {
+                        _logger.Trace("Replacing slashes in probable Windows UNC path '{0}'. If this is not meant to be a Windows UNC path fix your qBittorrent configuration", savePath);
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs:384]:
```suggestion
                        _logger.Trace("Replacing double back slashes in probable Windows UNC path '{0}'. If this is not meant to be a Windows UNC path fix your qBittorrent configuration and don't use // for paths on non-Windows systems in Qbit", savePath);
```

wordy but can likely tweak

Comment (markus101) [reply src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs:384]: Wrong slash, but definitely too wordy regardless.

Comment (markus101) [reply src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs]: I updated it to point them at the Save Path for the category, if they can't get there from them then there isn't much we can do to spell it out much more clearly.
